snapshots/smoking/2024-05-30/cigarette_sales.zip.dvc

```diff
@@ -0,0 +1,28 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: International Smoking Statistics
+    date_published: "2016-12-19"
+    title_snapshot: International Smoking Statistics - Cigarette Sales
+
+    # Citation
+    producer: Barbara Forey, Jan Hamling, John Hamling, Alison Thornton, Peter Lee
+    attribution: International Smoking Statistics (2016)
+    citation_full: |-
+      International Smoking Statistics WEB Edition
+      © Barbara Forey, Jan Hamling, John Hamling, Alison Thornton, Peter Lee, 2006-2016
+
+    # Files
+    url_main: http://www.pnlee.co.uk/ISS.htm
+    date_accessed: 2024-05-30
+
+    # License
+    license:
+      name: © Barbara Forey, Jan Hamling, John Hamling, Alison Thornton, Peter Lee, 2006-2016
```

Comment:
The license url should be the place where you find the copyright. In this case: http://pnlee.co.uk/ISS.htm